ppr-ui/src/components/userAccess/ReviewConfirm/ConfirmRequirements.vue

```diff
@@ -5,11 +5,14 @@
     :class="{'border-error-left': validateReview && !isRequirementsConfirmed}"
   >
     <template #formSlot>
-      <v-form class="pl-2">
-        <ListRequirements :requirements="requirements" />
+      <v-form>
+        <ListRequirements
+          class="pl-1"
+          :requirements="requirements"
+        />
         <v-checkbox
           v-model="isRequirementsConfirmed"
-          class="confirmation-checkbox mt-7"
+          class="confirmation-checkbox mt-7 ml-n4"
```

Comment:
I find it hard to work with negative margins. Messing up the styles, so had to refactor one of the components. I would try to avoid using them.